dev/io.openliberty.pages.4.0.internal.factories/src/org/apache/jasper/el/ELResolverImpl.java

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
Have you verified that this class is in fact ending up in the bundle? I know we've had some issues in the past with overlays showing up.

Reply:
<img width="712" alt="image" src="https://github.com/OpenLiberty/open-liberty/assets/5934310/a9775028-9048-4e42-8c85-1bfd9ea9cd8a">
Correct class is used since the`getFeatureDescriptors` method does not exist.